test/enhance/test_histogram.py

```diff
@@ -6,103 +7,178 @@
 from kornia.testing import assert_close
 
 
-class TestHistogram:
+class TestImageHist2d:
+    fcn = kornia.enhance.image_hist2d
+
+    @pytest.mark.parametrize("kernel", ["triangular", "gaussian", "uniform", "epanechnikov"])
+    def test_shape(self, device, dtype, kernel):
+        input = torch.ones(32, 32, device=device, dtype=dtype)
+        hist, pdf = TestImageHist2d.fcn(input, 0.0, 1.0, 256, kernel=kernel)
+        assert hist.shape == (1, 1, 256) and pdf.shape == (1, 1, 256)
+
+    @pytest.mark.parametrize("device", [("cuda"), ("cpu")])
```

Comment:
```suggestion
```
you don't need to add this line. It's done automatically during the testing setup